---
Header header
string[]   adapters # adapter names, eg. ["eth0", "wlo0"]
